phenol-core/src/main/java/module-info.java

```diff
@@ -0,0 +1,12 @@
+module org.monarchinitiative.phenol.core {
+  exports org.monarchinitiative.phenol.base;
+  exports org.monarchinitiative.phenol.graph;
+  exports org.monarchinitiative.phenol.ontology.data;
+  exports org.monarchinitiative.phenol.ontology.algo;
+  exports org.monarchinitiative.phenol.ontology.scoredist;
+  exports org.monarchinitiative.phenol.ontology.similarity;
+
+  requires transitive org.jgrapht.core; // due to DefaultDirectedGraph being exposed in MinimalOntology
+  requires com.google.common;
```

Comment:
Still using guava?

Reply:
After today's commits, it's gone forever! :confounded: